ksql-serde/src/test/java/io/confluent/ksql/serde/util/SerdeProcessingLogMessageFactoryTest.java

```diff
@@ -25,33 +25,36 @@
 import static org.junit.Assert.assertThat;
 
 import com.google.common.collect.ImmutableList;
+import io.confluent.ksql.processing.log.ProcessingLogConfig;
 import io.confluent.ksql.processing.log.ProcessingLogMessageSchema;
 import io.confluent.ksql.processing.log.ProcessingLogMessageSchema.MessageType;
 import io.confluent.ksql.serde.util.SerdeProcessingLogMessageFactory;
 import java.util.Base64;
+import java.util.Collections;
 import java.util.Optional;
 import java.util.stream.IntStream;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaAndValue;
 import org.apache.kafka.connect.data.Struct;
+import org.junit.After;
```

Comment:
nit: unused import. (Also a few more in this file, and in `EngineProcessingLogMessageFactoryTest.java` and `PhysicalPlanBuilderTest.java`